php/class-media.php

```diff
@@ -1230,11 +1230,25 @@ public function apply_default_transformations( array $transformations, $attachme
 		}
 		// Base image level.
 		$new_transformations = array(
+			'video'  => array(),
 			'image'  => Api::generate_transformation_string( $transformations, $type ),
 			'tax'    => array(),
 			'global' => array(),
 			'qf'     => array(),
 		);
+		// Adaptive streaming eagers.
+		if ( 'video' === $type ) {
+			$streaming = $this->settings->get_value( 'adaptive_streaming', 'adaptive_streaming_mode' );
+			if ( 'on' === $streaming['adaptive_streaming'] ) {
+				$streaming_transform          = array(
+					array(
+						'streaming_profile' => 'auto',
+						'fetch_format'      => $streaming['adaptive_streaming_mode'],
```

Comment:
<!-- highlight specific problems -->
**❗️issue:** 

@DavidCramer I think this is where I'm getting an error when syncing new videos, or trying to play videos that have been published before this code base.

The error is:
`streaming_profile must be the only directive in the transformation component` 

Can you please have a look here?

Reply:
fetch_format should not be added to the transformation, but rather the URL extension should be changed - .m3ud for HLS and .mpd for MPEG-DASH